packages/electron-chrome-web-store/src/browser/api.ts

```diff
@@ -50,6 +50,11 @@ function getExtensionInstallStatus(
   extensionId: ExtensionId,
   manifest?: chrome.runtime.Manifest,
 ) {
+  const customStatus = state.overrideExtensionInstallStatus?.(state, extensionId, manifest)
+  if (customStatus) {
```

Comment:
Just to be safe in case another type is returned.
```suggestion
  const customStatus: unknown = state.overrideExtensionInstallStatus?.(state, extensionId, manifest)
  if (typeof customStatus === 'string') {
```